test/unit/processing/get-payment-requests-from-file.test.js

```diff
@@ -0,0 +1,224 @@
+
+    batchHeaders = [{
+      batchValue: 200,
+      exportDate: '2021-08-12',
+      ledger: AP,
+      numberOfPaymentRequests: 2,
+      sequence: 1,
+      sourceSystem: 'SFIP'
+    }]
+
+    batchPaymentRequestsSFI = [{
+      contractNumber: 'SFIP000001',
+      currency: GBP,
+      deliveryBody: 'RP00',
+      frn: '1000000001',
+      invoiceNumber: 'SFI00000001',
+      paymentRequestNumber: 1,
+      schedule: M12,
+      value: 100,
+      invoiceLines: [{
+        accountCode: 'SOS27',
+        agreementNumber: 'SIP00000000001',
+        deliveryBody: 'RP00',
+        description: 'G00 - Gross value of claim',
+        dueDate: '2022-12-01',
+        fundCode: 'DRD10',
+        invoiceNumber: 'SFI00000001',
+        marketingYear: 2022,
+        schemeCode: '80001',
+        value: 100
+      }]
+    },
+    {
+      contractNumber: 'SFIP000002',
+      currency: GBP,
+      deliveryBody: 'RP00',
+      frn: '1000000002',
+      invoiceNumber: 'SFI00000002',
+      paymentRequestNumber: 3,
+      schedule: M12,
+      value: 100,
+      invoiceLines: [{
+        accountCode: 'SOS273',
+        agreementNumber: 'SIP00000000002',
+        deliveryBody: 'RP00',
+        description: 'G00 - Gross value of claim',
+        dueDate: '2022-12-01',
+        fundCode: 'DRD10',
+        invoiceNumber: 'SFI00000002',
+        marketingYear: 2022,
+        schemeCode: '80001',
+        value: 100
+      }]
+    }]
+
+    batchPaymentRequestsLumpSums = [{
+      contractNumber: 'L0000001',
+      currency: GBP,
+      deliveryBody: 'RP00',
+      frn: '1000000001',
+      invoiceNumber: 'LSES0000001',
+      paymentRequestNumber: 1,
+      value: 100,
+      invoiceLines: [{
+        deliveryBody: 'RP00',
+        description: 'G00 - Gross value of claim',
+        dueDate: '2022-12-01',
+        fundCode: 'DOM10',
+        invoiceNumber: 'LSES0000001',
+        marketingYear: 2022,
+        schemeCode: '10570',
+        value: 100
+      }]
+    },
+    {
+      contractNumber: 'L0000002',
+      currency: GBP,
+      deliveryBody: 'RP00',
+      frn: '1000000002',
+      invoiceNumber: 'LSES0000002',
+      paymentRequestNumber: 2,
+      value: 100,
+      invoiceLines: [{
+        deliveryBody: 'RP00',
+        description: 'G00 - Gross value of claim',
+        dueDate: '2022-12-01',
+        fundCode: 'DOM10',
+        invoiceNumber: 'LSES0000002',
+        marketingYear: 2022,
+        schemeCode: '10570',
+        value: 100
+      }]
+    }]
+
+    validateBatch.mockReturnValue(true)
+  })
+
+  afterEach(async () => {
+    jest.resetAllMocks()
+  })
+
+  test('should call validate when valid fileBuffer and sequence are received', async () => {
```

Comment:
Test names in this file are not matching the test scenario exactly

Reply:
Good spot.  Fixed.